AUTHORS

```diff
@@ -0,0 +1,12 @@
+# This is the official list of authors for copyright purposes.
+# This file is distinct from the CONTRIBUTORS files.
+# See the latter for an explanation.
+
```

Comment:
Please add this paragraph:

```
# This does not necessarily list everyone who has contributed code, since in
# some cases, their employer may be the copyright holder.  To see the full list
# of contributors, see the revision history in source control.
```

Reply:
Done with https://github.com/tensorflow/magenta-demos/pull/64/commits/415bc276d5d9bcb40a65bcdc1b4ec37dfa27a231